### PR TITLE
parallel e2e test runs

### PR DESCRIPTION
**Pull request by JKBGIT1**
closes https://github.com/berops/infra/issues/591

I tried to run two e2e tests in parallel. It worked, but I hit the quotas. Most of them were increased, but I can't guarantee you won't hit others. Also, Genesis Cloud approved new quotas, but apparently, they haven't been applied yet, because I hit them again while testing.

### CODE REVIEW

File: .github/workflows/CI-pipeline.yml [hunk 1]
```diff
@@ -334,9 +334,17 @@ jobs:
       - name: Start the E2E tests
         working-directory: ./manifests
         run: |
+          sudo apt update && sudo apt install -y wget tar
+          wget -q https://github.com/mikefarah/yq/releases/download/v4.27.2/yq_linux_amd64.tar.gz -O - |\
+          tar xz && mv yq_linux_amd64 yq
+
+          NAME_HASH="testing-framework-${SHORT_GITHUB_SHA}-${GITHUB_RUN_NUMBER}" ./yq e -i '(select(.kind == "ClusterRoleBinding").metadata.name = strenv(NAME_HASH))' ./testing-framework/testing-framework.yaml
```
Comment (cloudziu) [inline .github/workflows/CI-pipeline.yml:341]: Isn't the https://github.com/berops/claudie/blob/a7005d5e24e6cbaf6f98f8dd7bf628b6958a4786/.github/workflows/CI-pipeline.yml#L268 doing the same thing?

**Review by cloudziu — APPROVED** [review]
Thanks for the PR 😄 left one comment.

File: .github/workflows/CI-pipeline.yml (hunk 1 above)
Comment (JKBGIT1) [reply .github/workflows/CI-pipeline.yml:341]: yes, exactly.

**Review by Despire — APPROVED** [review]
👍

**Pull request merged**